model/docs/getting-started.qmd

```diff
@@ -0,0 +1,185 @@
+---
+title: "Getting started with pyrenew"
+format: gfm
+engine: knitr
```

Comment:
Do we definitely want a `knitr` rather than `jupyter` engine binding? As far as I can tell from [the docs](https://quarto.org/docs/computations/execution-options.html#engine-binding), using `knitr` as our engine will mean the Python execution happens via `reticulate`. If we don't specify an engine, Quarto should [default to `jupyter` given the presence of only Python code in the `.qmd`](https://quarto.org/docs/computations/execution-options.html#engine-binding). We could also explicitly specify `engine: jupyter`.

Reply:
I think it makes sense using jupyter. I'll switch to that.

Reply:
Yes, Jupyter is faster than knitr.